namelayer-spigot/src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: NameLayer
 main: vg.civcraft.mc.namelayer.NameLayerPlugin
-version: ${project.version}
+version: ${releaseVersion}
```

Comment:
The parent POM's version was fixed at 1.0.0 to never have to update that dependency in the child POMs. The child POMs still had their own versions. Does this move things back to requiring a version number update in multiple POMs?